src/Main.ts

```diff
@@ -277,6 +280,16 @@ export class Main {
             labels: ["outcome"],
             name: "remote_request_seconds",
         });
+        this.metrics.addGauge({
+            help: "count of active users",
+            labels: ["remote", "puppeted"],
```

Comment:
For the "remote" label, I want this to be true or false. This would give us both gauges.
```
bridge_active_users{remote=false} 34
bridge_active_users{remote=true} 27
```

Good call for pointing out we're interested in the number of puppets users, regardless of them being active or not. My next gauges would then be:
```
bridge_user_puppets{remote=false} 0
bridge_user_puppets{remote=true} 12
```

Reply:
Yep, makes sense to me. One last thing is that Slack users cannot puppet, so you can omit the remote field for the puppets metric :)